### PR TITLE
[DNM] Std migration for all

### PR DESCRIPTION
**Pull request by milchakov**

### CODE REVIEW

**Comment by milchakov** [comment]
DUDE JTTP

**Comment by milchakov** [comment]
DUDE JTTP

File: android/jni/com/mapswithme/maps/Framework.cpp [hunk 1]
```diff
@@ -324,7 +324,7 @@ m2::PointD Framework::GetViewportCenter() const
   return m_work.GetViewportCenter();
 }
 
-void Framework::AddString(string const & name, string const & value)
+void Framework::AddString(std::string const & name, std::string const & value)
```
Comment (ygorshenin) [inline android/jni/com/mapswithme/maps/Framework.cpp]: using namespace std at the top of the file will significantly reduce the number of changes.

Comment (milchakov) [inline android/jni/com/mapswithme/maps/Framework.cpp]: я поставил ограничение в 30 упоминаний пространства имен std. В этом файле добавлено 19. Возможно надо было поставить ограничение в 20.

Comment (ygorshenin) [inline android/jni/com/mapswithme/maps/Framework.cpp]: Так а зачем? Если раньше инклудился std/string.hpp, то там все было с using, поэтому это было эквивалентно #include <string> и using namespace std.

**Comment by milchakov** [comment]
DUDE JTTP

**Comment by milchakov** [comment]
DUDE JTTP

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by mapsmetest** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**